### PR TITLE
Update readmes and contributing page

### PR DESCRIPTION
**Pull request by bjohansebas**
Run `./get-readmes.sh` and `./get-contributing.sh`

In the future, it might be possible to remove external README files with links to the repositories themselves to avoid having to run that script.

### CODE REVIEW

**Review by UlisesGascon — APPROVED** [review]

**Comment by UlisesGascon** [comment]
What about moving this a GitHub action? So we can get an automated PR every few days? :thinking:

File: _includes/readmes/cookie-session.md [hunk 1]
```diff
@@ -103,6 +103,8 @@ The options can also contain any of the following (for the full list, see
   - `expires`: a `Date` object indicating the cookie's expiration date (expires at the end of session by default).
   - `path`: a string indicating the path of the cookie (`/` by default).
   - `domain`: a string indicating the domain of the cookie (no default).
+  - `partitioned`: a boolean indicating whether to partition the cookie in Chrome for the [CHIPS Update](https://developers.google.com/privacy-sandbox/3pcd/chips) (`false` by default). If this is true, Cookies from embedded sites will be partitioned and only readable from the same top level site from which it was created.
```
Comment (crandmck) [inline _includes/readmes/cookie-session.md:106]: sp: Boolean is always capitalized.  
"Cookies" should not be capitalized here.
Hyphenate "top-level".

**Review by crandmck — APPROVED** [review]

**Comment by crandmck** [comment]
After I made my minor comment, I realized that this text is actually in the https://github.com/expressjs/cookie-session README, so you can ignore my comment.  Some day, we need to make a comprehensive copy edit of all the middleware READMEs.

**Comment by crandmck** [comment]
@bjohansebas Why is the `check-translation` test failing?

**Comment by bjohansebas** [comment]
Close in favor of #1559, which automates the process.

**Pull request closed without merging**